src/main/java/pl/javastart/sellegro/auction/Auction.java

```diff
@@ -1,10 +1,14 @@
 package pl.javastart.sellegro.auction;
 
+import javax.persistence.Entity;
+import javax.persistence.Id;
 import java.math.BigDecimal;
 import java.time.LocalDate;
 
+@Entity
 public class Auction {
 
+    @Id
     private Long id;
```

Comment:
Tutaj jest dobrze pamiętać o dodawniu `@GeneratedValue` z racji tego że przy ewentualnym dodawniu nie chcelibyśmy tego robić ręcznie. W tym zadaniu nie trzeba zrobić dodawania, dlatego nie jest to jakiś wielki błąd

Reply:
Nie dawałem tego, ponieważ widziałem, że w pliku draft.sql są podawane wartości pola id. Uznałem, że to będzie kolidowało.